tests/test_service_routes.py

```diff
@@ -331,6 +331,27 @@ def test_create_shopcart_item(self):
         #    new_shopcart_item["update_time"], "Update time not set"
         #)
 
+    def test_update_shopcart_item(self):
+        """ Update an existing shopcart item """
+        # create a shopcart item to update
+        test_shopcart_item = ShopcartItemFactory()
+        resp = self.app.post(
+            "/shopcartitems", json=test_shopcart_item.serialize(), content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # update the shopcart item
+        new_shopcart_item = resp.get_json()
+        new_shopcart_item["price"] = "50.00"
+        resp = self.app.put(
+            "/shopcartitems/{}".format(new_shopcart_item["id"]),
+            json=new_shopcart_item,
+            content_type="application/json",
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        updated_shopcart_item = resp.get_json()
+        self.assertEqual(updated_shopcart_item["category"], "unknown")
```

Comment:
and add here:

self.assertEqual(updated_shopcart_item["sku"], 1001)
self.assertEqual(updated_shopcart_item["name"], "item_1")
self.assertEqual(updated_shopcart_item["amount"], 4)